src/commons/utils/IntroductionHelper.ts

```diff
@@ -25,6 +25,10 @@ const generateSourceDocsLink = (sourceChapter: Chapter, sourceVariant: Variant)
     );
   }
 
+  if (sourceChapter === Chapter.HTML) {
+    return 'However, you have chosen HTML, which renders your code directly as a HTML document.';
```

Comment:
![image](https://user-images.githubusercontent.com/5585517/194615230-428a0a1d-bbcd-4e79-b21d-88c9555c8827.png)

I'm not very sure what this sentence is trying to tell me, or how it might help a beginner better understand what HTML is.

Reply:
Updated description:
![image](https://user-images.githubusercontent.com/54243224/194693347-3d769fe8-e0f3-4f96-a115-faf76659fb82.png)